pom.xml

```diff
@@ -283,6 +293,35 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
```

Comment:
need to add java version like this https://github.com/folio-org/mod-orders/blob/master/pom.xml#L423